README.md

```diff
@@ -123,6 +123,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Get a list of file changes with PR/Push](https://github.com/trilom/file-changes-action)
 - [Use private actions in any workflow](https://github.com/InVisionApp/private-action-loader)
 - [Label Your Issues Using the Issue's Contents](https://github.com/damccorm/tag-ur-it)
+- [Build and publish docker images to any registry using Kaniko](https://github.com/outillage/kaniko-action)
```

Comment:
This action looks like it would be more appropriate under the "Deployment" - "Docker" section below since it is utilizing a third party entity as opposed to strictly GitHub tools. Would you mind moving it to that section?

Reply:
Agreed :)

Reply:
Thanks for the review, moved it 🙂